### PR TITLE
Dnsmasq

### PR DESCRIPTION
**Pull request by ZeroChaos-**
convert from dhcpd to dnsmasq since dhcpd seems to have randomly broken and we should prefer dnsmasq anyway.

since this removes the dead dhcpd config file it must be merged with https://github.com/pwnieexpress/pwnix_base_cookbook/pull/299 to avoid trying to install a file which no longer exists.

### CODE REVIEW

**Comment by ZeroChaos-** [comment]
available as update pack called "dnsmasq" for testing.  I'm not sure this will fix MOB-114, but I sure hope it does

**Comment by ZeroChaos-** [comment]
works flawlessly on my nexus 9, I didn't have a tplink handy to test on my shield (so it was the victim, muhahahahahahaha)

File: scripts/evilap.sh [hunk 1]
```diff
@@ -202,11 +206,31 @@ f_karmaornot(){
   #Bring up interface
   ifconfig "${evilap_eth}" up 192.168.7.1 netmask 255.255.255.0
 
-  #Start DHCP server on ${evilap_eth}
-  if [ -d /var/lib/dhcp ] && [ ! -f /var/lib/dhcp/dhcpd.leases ]; then
-    touch /var/lib/dhcp/dhcpd.leases
+  dnsmasq_conf=$(mktemp -t dnsmasq.conf-XXXX)
+  cat << EOF > "$dnsmasq_conf"
```
Comment (tmossey) [inline scripts/evilap.sh]: Should we check if this exists before we write to it so we don't write over it every time?

Comment (tmossey) [inline scripts/evilap.sh]: Or is this a tmp conf?

Comment (ZeroChaos-) [inline scripts/evilap.sh]: tmp config, automatically generated every time.  I started using cli flags and it just got too hard to read.

Comment (tmossey) [inline scripts/evilap.sh]: Yea I see what's going on now.

Comment (ZeroChaos-) [inline scripts/evilap.sh]: "XXXX" is filled in with random characters by mktemp so it won't overwrite in any meaningful time.

Comment (tmossey) [inline scripts/evilap.sh]: :+1:

**Comment by dallaswinger** [comment]
the update pack works after I manually pulled in the changes from pwnieexpress/pwnix_base_cookbook#299 

tested evilap and it is working. I don't get the error described in MOB-114

**Comment by tmossey** [comment]
Merging this based on @dallaswinger testing along with https://github.com/pwnieexpress/pwnix_base_cookbook/pull/299 for further testing

**Pull request merged**